pages/auth-login.html

```diff
@@ -0,0 +1,218 @@
+          />
+        </div>
+        <ul class="gentle-nav-links ecom-nav-links">
+          <li>
+            <a class="link-none" href="auth-signup.html"> SIGN UP </a>
+          </li>
+          <li>
+            <a class="link-none" href="cart-management.html">
+              <div class="badge mx-2">
+                <i class="fas fa-cart-plus fa-2x"></i>
+                <div class="badge-number gentle-flex-center">
+                  <span>2</span>
+                </div>
+              </div>
+            </a>
+          </li>
+          <li>
+            <a class="link-none" href="wishlist.html">
+              <div class="badge mx-2">
+                <i class="fas fa-heart fa-2x"></i>
+                <div class="badge-number gentle-flex-center">
+                  <span>2</span>
+                </div>
+              </div>
+            </a>
+          </li>
+        </ul>
+      </nav>
+      <!-- Navigation ends -->
+
+      <section class="main-content flex-column flex-center">
+        <div
+          class="main-content-container flex-column flex-justify-center p-10"
+        >
+          <div class="form-name text-center">
+            <h3>LOGIN</h3>
+          </div>
+          <form class="gentle-form-group gentle-flex-gap txt-sm">
+            <div class="gentle-input-group">
+              <label class="gentle-input-label">Email Address</label>
+              <input
+                type="email"
+                class="gentle-input"
+                placeholder="e.g. ashok_kamte@rediff.com"
+              />
+            </div>
+            <div class="gentle-input-group">
+              <label class="gentle-input-label">Password</label>
+              <input
+                type="password"
+                class="gentle-input"
+                placeholder="******"
+              />
+            </div>
+            <div
+              class="permission-check gentle-flex-gap flex-wrap flex-space-between flex-align-center"
+            >
+              <label
+                for="permission"
+                name="light"
+                class="gentle-flex-gap flex-align-center text-cursor-pointer checkbox-input-container mb-2"
+              >
+                <input
+                  type="checkbox"
+                  class="checkbox-input text-cursor-pointer"
+                  id="permission"
+                />
+                <span>Remember Me</span>
+              </label>
+              <div class="forget-password">
+                <a href="#" class="link-none"> Forgot Password? </a>
+              </div>
+            </div>
+            <div
+              class="submit-action gentle-flex-gap flex-column flex-justify-center"
+            >
+              <button
+                type="submit"
+                class="btn btn-info btn-info-hover submit-btn"
+              >
+                <a href="#" class="link-none"> LOG IN </a>
+              </button>
+              <button class="btn btn-link">
```

Comment:
Avoid adding `<a>` inside button, links may not clicked when you clicked on button.